examples/DotNetCore/ConsoleAppWithTableStorageReference/Program.cs

```diff
@@ -0,0 +1,182 @@
+﻿using Azure;
+using Azure.Data.AppConfiguration;
+using Azure.Data.Tables;
+using Azure.Identity;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Examples.ConsoleAppWithTableStorageReference
+{
+    // C# record type for items in the table
+    public record Product : ITableEntity
+    {
+        public string RowKey { get; set; } = default!;
+
+        public string PartitionKey { get; set; } = default!;
+
+        public string Name { get; init; } = default!;
+
+        public int Quantity { get; init; }
+
+        public bool OnSale { get; init; }
+
+        public ETag ETag { get; set; } = default!;
+
+        public DateTimeOffset? Timestamp { get; set; } = default!;
+    }
+
+    class Program
+    {
+        static IConfiguration Configuration { get; set; }
+
+        static void Main(string[] args)
+        {
+            Configure();
+
+            string display = string.Empty;
+
+            StringBuilder sb = BuildProductTable();
+
+            sb.AppendLine();
+            sb.AppendLine("Press any key to exit...");
+
+            display = sb.ToString();
+
+            Console.Clear();
+            Console.Write(display);            
+
+            // Finish on key press
+            Console.ReadKey();
+        }
+
+        private static StringBuilder BuildProductTable()
+        {
+            StringBuilder sb = new StringBuilder();
+
+            List<Product> tableContent = new List<Product>();
+            Configuration.GetSection("MyShop:Inventory").Bind(tableContent);
+
+            // Example value for MyShop:DisplayedColumns: Product OnSale
+            string myShopDisplayedColumns = Configuration.GetValue<string>("MyShop:DisplayedColumns");
```

Comment:
What do you think about storing this value as a list of strings with application/json content type? 
// Example value for MyShop:DisplayedColumns = [Product, OnSale]
// Content-Type = application/json